indexer/feature_utils.cpp

```diff
@@ -243,14 +267,26 @@ void GetReadableName(RegionData const & regionData, StringUtf8Multilang const &
 
   vector<int8_t> codes;
   // If MWM contains user's language.
-  if (regionData.HasLanguage(deviceLang))
+  bool const preferDefault = regionData.HasLanguage(deviceLang);
```

Comment:
Could you please explain why we prefer default language if region data contains device language? How device language and default language are related?